src/actions/dashboardActions.ts

```diff
@@ -34,6 +34,7 @@ export const SET_DASHBOARD = "SET_DASHBOARD"
 export const ADD_WIDGET = "ADD_WIDGET";
 export const REMOVE_WIDGET = "REMOVE_WIDGET";
 export const EDIT_WIDGET = "EDIT_WIDGET"
+export const MOVE_WIDGET = "MOVE_WIDGET"
```

Comment:
Curious about what the differences between ADD_WIDGET and MOVE_WIDGET are

Reply:
Added comments into the code